### PR TITLE
Fix #1066 -- Change installation tutorial to PostgreSQL

### LINKED ISSUE #1066: Change installation tutorial to show how to use PostgreSQL, not MySQL
PostgreSQL is our preferred database, so we should showcase it in our tutorials. To do this, you should try to install pretix following the tutorial, but with PostgreSQL, and change the tutorial where necessary.

https://docs.pretix.eu/en/latest/admin/installation/index.html

### PR DESCRIPTION
**Pull request by oocf**
This is the recommended database server so this documentation should use that

### CODE REVIEW

File: doc/admin/installation/docker_smallscale.rst [hunk 1]
```diff
@@ -58,12 +58,17 @@ Database
 --------
 
 Next, we need a database and a database user. We can create these with any kind of database managing tool or directly on
-our database's shell, e.g. for MySQL::
-
-    $ mysql -u root -p
-    mysql> CREATE DATABASE pretix DEFAULT CHARACTER SET utf8mb4 DEFAULT COLLATE utf8mb4_unicode_ci;
-    mysql> GRANT ALL PRIVILEGES ON pretix.* TO pretix@'localhost' IDENTIFIED BY '*********';
-    mysql> FLUSH PRIVILEGES;
+our database's shell, e.g. for POSTGRESQL::
```
Comment (rixx) [inline doc/admin/installation/docker_smallscale.rst]: Please don't use ALLCAPS here. I think the preferred style is "PostgreSQL".

File: doc/admin/installation/docker_smallscale.rst [hunk 2]
```diff
@@ -58,12 +58,17 @@ Database
...
+
+    $ psql postgres -U root
```
Comment (rixx) [inline doc/admin/installation/docker_smallscale.rst]: This seems like a convoluted way of creating a user and database – why are those "\du" lines included? Why do you need to sign out and then use postgres as the pretix user?

Either this:
```SQL
$ psql
postgres=# CREATE DATABASE yourdbname;
postgres=# CREATE USER youruser WITH ENCRYPTED PASSWORD 'yourpass';
postgres=# GRANT ALL PRIVILEGES ON DATABASE yourdbname TO youruser;
```
or even just this
```
sudo -u postgres -i
createuser -W <username>
createdb -O <username> <dbname>
```
should be sufficient, shouldn't it?

File: doc/admin/installation/docker_smallscale.rst [hunk 3]
```diff
@@ -114,13 +119,14 @@ Fill the configuration file ``/etc/pretix/pretix.cfg`` with the following conten
     datadir=/data
 
     [database]
-    ; Replace mysql with postgresql_psycopg2 for PostgreSQL
-    backend=mysql
+    ; Replace postgresql_psycopg2 with mysql for MySQL
+    backend=postgresql_psycopg2
```
Comment (raphaelm) [inline doc/admin/installation/docker_smallscale.rst]: I know, the comment is misleading, but ``postgresql_psycopg2`` is actually deprecated by now in Django, so we can just use ``postgresql`` here instead.

File: doc/admin/installation/manual_smallscale.rst [hunk 4]
```diff
@@ -50,12 +50,17 @@ Database
 --------
 
 Having the database server installed, we still need a database and a database user. We can create these with any kind
-of database managing tool or directly on our database's shell, e.g. for MySQL::
-
-    $ mysql -u root -p
-    mysql> CREATE DATABASE pretix DEFAULT CHARACTER SET utf8mb4 DEFAULT COLLATE utf8mb4_unicode_ci;
-    mysql> GRANT ALL PRIVILEGES ON pretix.* TO pretix@'localhost' IDENTIFIED BY '*********';
-    mysql> FLUSH PRIVILEGES;
+of database managing tool or directly on our database's shell, e.g. for POSTGRESQL:
```
Comment (raphaelm) [inline doc/admin/installation/manual_smallscale.rst]: This needs to be a double-colon (``::``) at the end of the line such that it is recognized as a code snippet.

**Comment by raphaelm** [comment]
Replaced by 7219575b

**Pull request closed without merging**